src/settings.ts

```diff
@@ -24,4 +24,7 @@ export class DebuggerSettings {
         return this.debugSettings.get<string>("gdbtty");
     }
 
+    public get cobcrunpath(): string {
```

Comment:
Do you use that new method somewhere?

Reply:
Yes. My point is I did not see a call to this method. I assume it needs to be called to get that path somewhere.

Reply:
Ok I think we are confused here. This file seems to be only for the `gdb.ts` to have access to the config (which is kinda weird because in other files the access is done directly via the `vscode` API), since the files I edited do not use this `DebuggerSettings` class, I did not updated it first.

The actual place where we add configuration options for the extension is in the `package.json` which is generated by `superbol-studio-oss`, so we do not see those edit here ....

Maybe a symlink between the two `package.json` to update the one commited here could be useful

Reply:
Yes `cobcrunpath` needs to be extracted elsewhere and passed here as an argument.